media/css/sandstone/sandstone-resp.less

```diff
@@ -166,7 +166,48 @@ a.more {
             }
         }
     }
+}
+
+// mozID visual style
+// full width rows w/differing bg colors
+// TODO: needs responsive styles
+.mozID {
+    a:link,
+    a:active,
+    a:focus,
+    a:visited {
+        color: #fff;
+
+        &:hover {
+            text-decoration: underline;
+        }
+    }
+
+    h1, h2, h3, h4, h5, h6, .huge, .large {
+        color: #fff;
+        text-shadow: none;
+    }
+
+    #wrapper {
+        width: 100%;
+        padding-top: 64px; // make room for #masthead
+
+        .container {
+            width: (@widthDesktop - (@baseLine * 2));
+            padding-left: @baseLine;
+            padding-right: @baseLine;
+        }
+    }
 
+    #masthead {
+        position: absolute;
```

Comment:
Doesn't necessarily warrant changing, but just curious why we're using absolute positioning for the masthead here, seeing as this takes the element out of the regular flow of the document. Is this for a special case?